std/container/rbtree.d

```diff
@@ -1094,6 +1094,110 @@ if (is(typeof(binaryFun!less(T.init, T.init))))
         assert(t1 != o);  // pathological case, must not crash
     }
 
+    /**
+     * Generates a hash for a Red Black Tree
+     *
+     *
+     */
+    override size_t toHash() nothrow @safe
+    {
+        import std.traits : hasMember;
+
+        auto thisRange = this[];
+
+        static if (hasMember!(Elem, "toHash"))
+        {
+            size_t hash = 0;
```

Comment:
The initializer is useless (although the official position is to keep them for readability).

Reply:
FWIW there's no official position on either position yet, I think it's more that PRs that remove them don't contain much value and imho sometimes in more complicated code it does help with readability.